misc/json-to-record-converter/build.gradle

```diff
@@ -0,0 +1,59 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+apply from: "$rootDir/gradle/javaProject.gradle"
+
+configurations {
+    compile.transitive = false
+    compileClasspath.extendsFrom(compileOnly)
+}
+
+dependencies {
+    compileOnly project(':ballerina-parser')
+    compileOnly project(':ballerina-lang')
+    compileOnly project(':formatter:formatter-core')
+
+    implementation project(':ballerina-tools-api')
+    implementation project(':identifier-util')
+    implementation "com.google.code.gson:gson:${project.gsonVersion}"
+    implementation "org.apache.commons:commons-lang3:${project.apacheCommonsLang3Version}"
+    implementation "org.javatuples:javatuples:${project.javaTuples}"
```

Comment:
Do we need this?

Reply:
JavaTuples are being used in finding the type when two fields are with same name and different values.

```java
io.ballerina.jsonmapper.JsonToRecordMapper
```

Line 251